hhvm.rb

```diff
@@ -141,6 +141,7 @@ def install
     # Dependency information.
     cmake_args += %W[
       -DAWK_EXECUTABLE=#{Formula["gawk"].opt_bin}/gawk
+      -DBoost_ROOT=#{Formula["homebrew/versions/boost160"]}
```

Comment:
If we are setting the Include and Library directories, is this needed?  Won't hurt, I am just curious.